samples/instanceadmin/instanceadmin.py

```diff
@@ -1,259 +1,270 @@
-                        help='ID of the Cloud Bigtable instance to \
-                        connect to.')
-    parser.add_argument('cluster_id',
-                        help='ID of the Cloud Bigtable cluster to \
-                        connect to.')
-
-    args = parser.parse_args()
-
-    if args.command.lower() == 'run':
-        run_instance_operations(args.project_id, args.instance_id)
-    elif args.command.lower() == 'dev-instance':
-        create_dev_instance(args.project_id, args.instance_id,
-                            args.cluster_id)
-    elif args.command.lower() == 'del-instance':
-        delete_instance(args.project_id, args.instance_id)
-    elif args.command.lower() == 'add-cluster':
-        add_cluster(args.project_id, args.instance_id, args.cluster_id)
-    elif args.command.lower() == 'del-cluster':
-        delete_cluster(args.project_id, args.instance_id, args.cluster_id)
-    else:
-        print('Command should be either run \n Use argument -h, \
-               --help to show help and exit.')
+#!/usr/bin/env python
+
+# Copyright 2018, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Demonstrates how to connect to Cloud Bigtable and run some basic operations.
+#     http://www.apache.org/licenses/LICENSE-2.0
+Prerequisites:
+- Create a Cloud Bigtable project.
+  https://cloud.google.com/bigtable/docs/
+- Set your Google Application Default Credentials.
+  https://developers.google.com/identity/protocols/application-default-credentials
+
+Operations performed:
+- Create a Cloud Bigtable Instance.
+- List Instance for a Cloud Bigtable.
+- Delete a Cloud Bigtable Instance.
+- Create a Cloud Bigtable Cluster.
+- List Cloud Bigtable Clusters.
+- Delete a Cloud Bigtable Cluster.
+"""
+
+import argparse
+
+from google.cloud import bigtable
+from google.cloud.bigtable import enums
+
+
+def run_instance_operations(project_id, instance_id, cluster_id):
+    """Check Instance exists.
+        Creates a Production instance with default Cluster.
+        List instances in a project.
+        List clusters in an instance.
+
+    :type project_id: str
+    :param project_id: Project id of the client.
+
+    :type instance_id: str
+    :param instance_id: Instance of the client.
+    """
+    client = bigtable.Client(project=project_id, admin=True)
+    location_id = "us-central1-f"
+    serve_nodes = 1
+    storage_type = enums.StorageType.SSD
+    production = enums.Instance.Type.PRODUCTION
```

Comment:
I'm not sure if the instance_type param is still needed since all are production

Reply:
a8f33ac